lib/ansible.py

```diff
@@ -0,0 +1,31 @@
+import os
+
+from . import util, versions
+
+
+def install_deps():
+    """
+    Download and install any external dependencies required for Ansible to run.
+    """
+    # RHEL has rhc-worker-playbook
+    if versions.rhel.is_true_rhel():
+        # RHEL-7 seems to have it bundled in the base RPM
+        if versions.rhel <= 7:
+            return
+        pkg = 'rhc-worker-playbook'
+        ret = util.subprocess_run(['rpm', '--quiet', '-q', pkg])
+        if ret.returncode != 0:
+            util.subprocess_run(
+                ['dnf', '-y', '--nogpgcheck', 'install', pkg],
+                check=True)
```

Comment:
Why not to leave it in `recommend` package?

Reply:
I guess that's a good point.

The `install_deps()` function can probably still remain there, so test code doesn't have to `if` based on distro type and version, but the RHEL section can just `rpm -q` check for `rhc-worker-playbook` to be installed and `RuntimeError()` if it is not.

Would that make sense?

Reply:
+1, makes sense.
I like the `rpm -q` check to get error right away. And not spending couple of minutes (e.g. on VM install) and then getting Ansible fatal error because of missing module.